task_5/Tatiana.Derevenets/.idea/compiler.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add the ".idea" folder to .gitignore file
Actually I have already added files which should be ignored into it, so you can copy from it
Also you should delete now this folder and make another commit locally and then push your changes again